OpenOversight/app/utils.py

```diff
@@ -29,6 +30,13 @@
 SAVED_UMASK = os.umask(0o077)
 
 
+def get_s3_connection(config=None):
+    if os.getenv("S3_ENDPOINT"):
```

Comment:
How would you feel about moving this into app/config.py and setting a constant there, so we are pulling all our env vars from one place? Could plop it right above https://github.com/lucyparsons/OpenOversight/blob/develop/OpenOversight/app/config.py#L31 and then do a `if current_app.config['S3_ENDPOINT']:` here (and in the endpoint_url setting on the next line)?